lib/package.json

```diff
@@ -40,14 +39,14 @@
     "eslint-plugin-cypress": "^2.13.3",
     "eslint-plugin-jest": "^27.2.2",
     "jest": "^29.5.0",
-    "react": "^17.0.2",
-    "react-dom": "^17.0.2",
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0",
     "start-server-and-test": "^2.0.0",
     "typescript": "^5.1.3",
     "vite": "^4.3.9"
   },
   "peerDependencies": {
-    "gatsby": "^3.0.0 || ^4.0.0"
+    "gatsby": "^3.0.0 || ^4.0.0 || ^5.0.0"
```

Comment:
As this package depends on `gatsy-source-filesystem@5.7.0` (for Gatsby 5) I think that would make it incompatible with Gatsby 4 - if this is correct, we should just keep `^5.0.0` in the `peerDependencies`

Reply:
Somehow, I have `^5.0.0` already included in my local and it seems it was already added last month @alexjoverm

Reply:
![Screenshot 2023-07-07 at 14 36 58](https://github.com/storyblok/gatsby-source-storyblok/assets/25793187/33fd5ba2-e42b-40e9-9c15-5560f0c84f08)

Reply:
Done @alexjoverm